lib.sh

```diff
@@ -167,13 +167,20 @@ SPAMHAUS=/etc/spamhaus.wl
 ENVASIVE=/etc/apache2/mods-available/mod-evasive.load
 APACHE2=/etc/apache2/apache2.conf
 # Full text Search
-es_install() {
+opensearch_install() {
     INDEX_USER=$(gen_passwd "$SHUF" '[:lower:]')
-    ROREST=$(gen_passwd "$SHUF" "A-Za-z0-9")
-    nc_fts="ark74/nc_fts"
-    fts_es_name="fts_esror"
-}
-[ -n "$ES_INSTALL" ] && es_install # TODO: remove this line someday
+    OPNSREST=$(gen_passwd "$SHUF" "A-Za-z0-9")
+    opens_fts="opensearchproject/opensearch"
+    fts_node="fts_os-node"
+}
+create_certs(){
+    rm "$OPNSDIR"/"$1"
+    curl -s "$STATIC"/"$1" > "$OPNSDIR"/"$1"
+    sed -i "s|__NCDOMAIN__|$NCDOMAIN|" "$OPNSDIR"/"$1"
+    cd "$OPNSDIR"
+    bash "$1"
+}
+[ -n "$opensearch_install" ] && opensearch_install # TODO: remove this line someday
```

Comment:
I think we can get rid of this line finally

Reply:
done